examples/secured-client-with-jwt-auth/secured_client_with_jwt_auth.description

```diff
@@ -0,0 +1,5 @@
+// A secured client with jwt authentication is used to connect to a
+// secured service with jwt authentication. An additional `auth`
+// field is added to the HTTP client endpoint initialization in order
+// to make the simple HTTP client endpoint into a secure client endpoint.
+// Inside the `auth` filed the `scheme` is defined for JWT authentication.
```

Comment:
Same typo as before. Fix in other similar places.

Reply:
Fixed.